pvfactors/engine.py

```diff
@@ -14,9 +13,8 @@ class PVEngine(object):
     as a timeseries when the pvarrays can be build from dictionary parameters
     """
 
-    def __init__(self, params, vf_calculator=VFCalculator(),
+    def __init__(self, pvarray, vf_calculator=VFCalculator(),
```

Comment:
it's meaningless to have ``pvarray=OrderedPVArray()``, because it will make something fail later.
It's better to initialize the PV array outside of the engine and then pass it in